AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.html

```diff
@@ -25,7 +25,22 @@
     </div>
   </div>
 
-  <div class="participant-row" *ngIf="!isJudge">
+  <div class="staff-member-row" *ngIf="isStaffMember">
+    <div class="wrap staff-member-display-name">
+      {{ participant.display_name }}
+      <div>
+        <strong>{{ staffMemberRole }}</strong>
+      </div>
+      <div class="staff-member-row__email" id="staff-member-email">{{ participant.email }}</div>
+    </div>
+
+    <div *ngIf="canEdit" class="participant-row__actions">
+      <a class="vhlink" [routerLink]="['/assign-judge']">Edit</a>
```

Comment:
Please add an ID to this element so that Automation can reference this for testing